Fabric.Identity.API/scripts/tests/testInstall.config

```diff
@@ -0,0 +1,110 @@
+      <!-- The thumb-print of an SSL certificate to use for signing access and identity tokens. 
+           For initial installation it is recommended to use the thumb-print of the certificate 
+           providing TLS/SSL for the website. -->
+      <variable name="iisUser" value="" />
+      <variable name="primarySigningCertificateThumbprint" value="" />
+      <!-- The thumb-print of an SSL certificate to use for encrypting/decrypting sensitive information 
+           in the config. For initial installation it is recommended to use the thumb-print of the 
+           certificate providing TLS/SSL for the website.-->
+      <variable name="encryptionCertificateThumbprint" value="" />
+      <!-- An Azure Application Insights instrumentation key that allows telemetry to be recorded in 
+           Azure Application Insights. If no instrumentation key is provided, no telemetry will be sent. -->
+      <variable name="appInsightsInstrumentationKey" value="" />
+      <!-- The name of the IIS website to install the application under -->
+      <variable name="siteName" value="Default Web Site" />
+      <!-- The name of the Identity database on the sql server instance -->
+      <variable name="identityDbName" value="Identity" />
+      <!-- The Sql Server role that the app pool user will be added to -->
+      <variable name="identityDatabaseRole" value="IdentityServiceRole" />
+      <!-- Enables Azure AD integration -->
+      <variable name="useAzureAD" value="False" />
+      <!-- Enables Windows AD integration -->
+      <variable name="useWindowsAD" value="True" />
+      <!-- Tenants identity is to be registered with -->
+      <tenants>
+        <variable name="4d07d6d8-58e4-45a4-8ce9-5d2cfc00c65f" alias="MikeTenant" />
+        <variable name="0ced5a42-fa6e-4d17-bb75-da8c4dbec43c" alias="RyanTenant" />
+      </tenants>
+      <!-- Reply Url for Azure App registration -->
+      <replyUrls>
+        <variable name="https://hckeniis2016.hqcatalyst.local/identity" alias="" />
+        <variable name="https://hckeniis2016.hqcatalyst.local/identity/signin-oidc" alias="" />
+      </replyUrls>
+      <!-- The Tenant identity is to be registered with -->
+      <claimsIssuerTenant>
+        <variable name="4d07d6d8-58e4-45a4-8ce9-5d2cfc00c65f" alias="MikeTenant" />
+      </claimsIssuerTenant>
+      <!-- The Tenants that are allowed through to the claims issuer  -->
+      <allowedTenants>
+        <variable name="0ced5a42-fa6e-4d17-bb75-da8c4dbec43c" alias="RyanTenant" />
+      </allowedTenants>
+      <!-- The name of the secret to be created for the azure application registration -->
+      <variable name="azureSecretName" value="AADOnceAgain" />
+      <registeredApplications>
+        <variable appName="Identity Provider Search Service" tenantId="4d07d6d8-58e4-45a4-8ce9-5d2cfc00c65f" tenantAlias="MikeTenant" clientid="2057a221-b20a-4491-9170-9638883e6af0" secret="tTSDjz3ksRYXtDRilAWuzdjIwQ8Sd1kjI2H2H8NtQwU=" />
```

Comment:
Swap these with dummy values instead of actual values

Reply:
yup